sorts/timsort.py

```diff
@@ -73,10 +73,14 @@ def timsort(lst):
 
 
 def main():
-
-    lst = [5,9,10,3,-4,5,178,92,46,-18,0,7]
+    assert timsort([]) == []
```

Comment:
Could we do:

```python
test_lists = [
  [],
  [1],
  [5, 9, 10, 3, -4, 5, 178, 92, 46, -18, 0, 7],
  [1, 2, 5, 8, 3, 6, 9, 4, 7],
]

for test_list in test_lists:
  assert timsort(test_list) == sorted(test_list)
```

It'll be easier to add test cases later, and is less error-prone (avoids copy-paste errors)